tests/e2e/tidbcluster/serial.go

```diff
@@ -825,6 +858,30 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 			err = mock.SetPrometheusResponse(monitor.Name, monitor.Namespace, mp, fw)
 			framework.ExpectNoError(err, "set tikv mock metrics error")
 
+			err = wait.Poll(10*time.Second, 5*time.Minute, func() (done bool, err error) {
+				tac, err = cli.PingcapV1alpha1().TidbClusterAutoScalers(ns).Get(tac.Name, metav1.GetOptions{})
+				if err != nil {
+					return false, nil
+				}
+				if tc.Annotations == nil || len(tc.Annotations) < 1 {
+					klog.Infof("tc haven't marked any annotation")
+					return false, nil
+				}
```

Comment:
Why we need to check the tc annotations here? It seems the e2e tesd failed due to this.